docs/src/pages/docs/functions/combinators.md

```diff
@@ -130,9 +237,64 @@ flip :: (a -> b -> c) -> b -> a -> c
 ```
 
 This little function just takes a function and returns a function that takes
-the first two parameters in reverse. One can compose flip calls down the line to
-flip all, or some of the other parameters if there are more than two. Mix and
-match to your heart's desire.
+the first two parameters in reverse. `flip` is perfectly suited for those
+moments where you have the context of your function but not the data. 
+Applying `flip` to the function will allow you to pass in your context and will
+return a function waiting for the data. This will happen often when you're
+using composition.
+
+When required, one can compose flip calls down the line to flip all, or some of
+the other parameters if there are more than two. Mix and match to your heart's
+desire.
+
+```javascript
+import flip from 'crocks/combinators/flip'
+
+import Pred from 'crocks/Pred'
+
+import composeB from 'crocks/combinators/composeB'
+import concat from 'crocks/pointfree/concat'
+import isNumber from 'crocks/predicates/isNumber'
+import mconcat from 'crocks/helpers/mconcat'
+import runWith from 'crocks/pointfree/runWith'
+
+concat('first param. ', 'second param. ')
+//=> "second param. first param. ""
+
+flip(concat, 'first param. ', 'second param. ')
+//=> "first param. second param. ""
+
+// checkAll :: [ a -> Boolean ] -> a -> Boolean
+const checkAll =
+  composeB(flip(runWith), mconcat(Pred))
+
+// lte :: Number -> Number -> Number
```

Comment:
So sorry looks like I missed a couple things on that initial pass. This siggy should be:
```js
// lte :: Number -> Number -> Boolean
```